@3rdweb-sdk/react/hooks/usePayments.ts

```diff
@@ -862,3 +889,181 @@ export function usePaymentsSellerByAccountId(accountId: string) {
+  invariant(accountId, "accountId is required");
+  const queryClient = useQueryClient();
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+
+  const [createWebhookBySellerId] = useInsertWebhookMutation({
+    refetchQueries: [WebhooksBySellerIdDocument],
+  });
+
+  return useMutationWithInvalidate(
+    async (input: CreateWebhookInput) => {
+      invariant(address, "No wallet address found");
+      invariant(paymentsSellerId, "No seller id found");
+      invariant(isValidWebhookUrl(input.url), "Invalid webhook url");
+
+      const webhookInput: Webhook_Insert_Input = {
+        seller_id: paymentsSellerId,
+        url: input.url,
+        is_production: input.isProduction,
+      };
+
+      return createWebhookBySellerId({
+        variables: {
+          object: webhookInput,
+        } as InsertWebhookMutationVariables,
+      });
+    },
+    {
+      onSuccess: () => {
+        return queryClient.invalidateQueries(paymentsKeys.webhooks(accountId));
+      },
+    },
+  );
+}
+
+export type UpdateWebhookInput = {
+  webhookId: string;
+  url?: string;
+  deletedAt?: Date;
+};
+
+export function usePaymentsUpdateWebhook(accountId: string) {
+  invariant(accountId, "accountId is required");
+  const queryClient = useQueryClient();
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+
+  const [updateWebhookBySellerId] = useUpdateWebhookMutation({
+    refetchQueries: [WebhooksBySellerIdDocument],
+  });
+
+  return useMutationWithInvalidate(
+    async (input: UpdateWebhookInput) => {
+      invariant(address, "No wallet address found");
+      invariant(paymentsSellerId, "No seller id found");
+
+      return updateWebhookBySellerId({
+        variables: {
+          id: input.webhookId,
+          webhookValue: {
+            url: input.url,
+            deleted_at: input.deletedAt,
+          } as Webhook_Set_Input,
+        } as UpdateWebhookMutationVariables,
+      });
+    },
+    {
+      onSuccess: () => {
+        return queryClient.invalidateQueries(paymentsKeys.webhooks(accountId));
+      },
+    },
+  );
+}
+
+export type PaymentsWebhookSecretType = {
+  id: string;
+  ownerId: string;
+  createdAt: string;
+  hashedKey: string;
+};
+
+export function usePaymentsWebhooksSecretKeyByAccountId(accountId: string) {
+  invariant(accountId, "accountId is required");
+
+  const fetchFromPaymentsAPI = usePaymentsApi();
+
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+
+  return useQuery(
+    paymentsKeys.webhookSecret(accountId),
+    async () => {
+      invariant(address, "No wallet address found");
+
+      return fetchFromPaymentsAPI(
+        "POST",
+        "api-key/get-decrypted-key",
+        {
+          sellerId: paymentsSellerId,
+        },
+        {
+          isSellerApiKey: true,
+        },
+      );
+    },
+    { enabled: !!paymentsSellerId && !!address },
```

Comment:
```suggestion
    { enabled: !!paymentsSellerId && !!accountId },
```

and remove useAddress